tests/test_leb/test_fast_calc.py

```
@@ -270,22 +270,48 @@ def test_truepos_flag(self, leb_reader, jd_mid):
-        """SEFLG_XYZ should raise KeyError (fall back to Skyfield)."""
-        with pytest.raises(KeyError, match="SEFLG_XYZ"):
-            fast_calc_ut(leb_reader, jd_mid, SE_SUN, SEFLG_XYZ)
+    def test_topoctr_with_topo(self, leb_reader, jd_mid):
+        """SEFLG_TOPOCTR works after swe_set_topo()."""
+        import libephemeris
+        from libephemeris import state
+
+        saved = state._TOPO
+        libephemeris.set_topo(12.5, 41.9, 0)
+        try:
+            result, flags = fast_calc_ut(leb_reader, jd_mid, SE_SUN, SEFLG_TOPOCTR)
```

Ruff (0.15.12)

[warning] Unpacked variable `flags` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)